src/fonduer/parser/visual_linker.py

```diff
@@ -160,7 +163,7 @@ def _coordinates_from_HTML(
                     xmax = int(float(word.get("xmax")))
                     for content in self.separators.split(word.getText()):
                         if len(content) > 0:  # Ignore empty characters
-                            word_id = (page_num, i)
+                            word_id: WordId = (page_num, i)
```

Comment:
This is a good explanation of WordId.

Reply:
I believe this was resolved by a2bcb86a94f2a00d7a9bbaad73f96973f1c4fd14.